packages/experiments/src/components/LayoutGroup/LayoutGroup.props.ts

```diff
@@ -3,18 +3,18 @@ import * as React from 'react';
 import { IBaseProps } from 'office-ui-fabric-react/lib/Utilities';
 import { LayoutGroup } from './LayoutGroup';
 
-export interface ILayoutGroupProps extends IBaseProps, React.HTMLAttributes<LayoutGroup> {
+export interface ILayoutGroupProps extends IBaseProps, React.HTMLAttributes<LayoutGroup | HTMLDivElement> {
   /**
    * Direction in which the child elements will be layed out.
    * @default vertical
    */
   direction?: 'vertical' | 'horizontal';
 
   /**
-   * Number of pixels between each element. No gap applied to last child.
+   * Number of pixels between each element. No layoutGap applied to last child.
    * @default 8
    */
-  gap?: number;
+  layoutGap?: number;
```

Comment:
i'm not sure layoutGap makes more sense; could we maybe consider removing this and replacing with the whole `layoutType=formFields|2column|3column|buttons`?

The real reason is that we can very much abstract basic layout concepts beyond what flexbox can do. Instead of having the consuming provide "8px gap space", they say "I'm rendering buttons" and we can infer the gap space (and control it universally.)

It also opens up opportunities for flexing in smaller screen widths. Maybe in mobile, form fields should have smaller gaps.

In the end this may mean that layout is a bit more rigid, but much more controllable and predictable for the consumer. We should use it in product, find the use cases we run across and make sure we have enough profiles that it's robust but not bloated.

Reply:
since this is just experiments, I'm going to merge this in as is. We can tackle how we want to abstract this before pulling this into OUFR.